fs/remote/blob.go

```diff
@@ -151,6 +153,24 @@ func (b *blob) FetchedSize() int64 {
 	return sz
 }
 
+func (b *blob) fetch(offset int64, size int64, fr *fetcher, cacheOpts *options) error {
```

Comment:
The function name should be `cache`.

Reply:
`blob` has a field named `cache` too. 

The build fails with following error:

```
fs/remote/blob.go:156:6: type blob has both field and method named cache
make: *** [containerd-stargz-grpc] Error 2
```